plugins/modules/redhat_subscription.py

```diff
@@ -923,6 +922,8 @@ def main():
                 else:
                     module.exit_json(changed=False, msg="System already registered.")
         else:
```

Comment:
Per this `if` block, I believe that  `force_register` must have a `required_if` for those same parameters.